front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -16,7 +18,7 @@ const slideInOutEasing = 'cubic-bezier(0.19, 1, 0.22, 1)';
 
 const Container = styled.div<{
   windowHeight: number;
-  windowWidth?: number;
+  windowWidth: number;
```

Comment:
Unrelated to the fix. I just saw I forgot to pass this value without a good reason. Checked layouts afterwards.